gymnasium/dev_wrappers/lambda_action.py

```diff
@@ -28,3 +35,152 @@ def __init__(
     def action(self, action: ActType) -> Any:
         """Apply function to action."""
         return self.func(action)
+
+
+class LambdaCompositeActionV0(LambdaActionV0):
```

Comment:
@RedTachyon Are you sure that this is the best way? Will users use this / does it provide a helpful abstraction that we will actually use?